### ISSUE TITLE
Still request failed with status code 429 and connections reset

### ISSUE THREAD

**Issue by TomKrysz** [apatsufas/homebridge-tapo-p100#35]
**Describe The Bug:**
Versions 16 and 17 beta, still with bug 449. Status requests seem to overlap with timeouted reconnections, resulting in tcp L510 flooding and connection dropping. Several hours after restarting HB, four L510s are plagued by several dozen connections per second, which generates error 449 (checked in router logs). Please do something about it, as plugins from version 16 on are useless as they are.

**To Reproduce:**
Just start/reboot HB service, wait a few minutes and watch the rapidly increasing HB tcp port 80 connections to L510.

**Expected behavior:**
Elimination of overlapping connections (summing reconnections, and the set frequency of device state requests).

**Logs:**
Logs, as in my previous post.

**Comment by TomKrysz** [comment]
[05/04/2022, 07:59:38] [Tapo Smart Platform] 371 Error: read ECONNRESET
[05/04/2022, 07:59:38] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/04/2022, 07:59:38] [Tapo Smart Platform] 371 Error: read ECONNRESET
[05/04/2022, 07:59:38] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/04/2022, 07:59:38] [Tapo Smart Platform] 371 Error: read ECONNRESET
[05/04/2022, 07:59:38] [Tapo Smart Platform] 371 Error: read ECONNRESET
[05/04/2022, 07:59:38] [Tapo Smart Platform] 371 Error: read ECONNRESET
[05/04/2022, 07:59:38] [Tapo Smart Platform] 371 Error: read ECONNRESET
[05/04/2022, 07:59:38] [Tapo Smart Platform] 371 Error: read ECONNRESET
[05/04/2022, 07:59:38] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/04/2022, 07:59:38] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/04/2022, 07:59:38] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/04/2022, 07:59:38] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/04/2022, 07:59:39] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/04/2022, 07:59:39] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/04/2022, 07:59:39] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/04/2022, 07:59:40] [Tapo Smart Platform] 371 Error: read ECONNRESET
[05/04/2022, 07:59:40] [Tapo Smart Platform] 371 Error: read ECONNRESET
[05/04/2022, 07:59:40] [Tapo Smart Platform] 371 Error: read ECONNRESET
[05/04/2022, 07:59:40] [Tapo Smart Platform] 371 Error: read ECONNRESET
[05/04/2022, 07:59:40] [Tapo Smart Platform] 371 Error: read ECONNRESET
[05/04/2022, 07:59:40] [Tapo Smart Platform] 371 Error: read ECONNRESET
[05/04/2022, 07:59:40] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/04/2022, 07:59:40] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/04/2022, 07:59:42] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/04/2022, 07:59:42] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/04/2022, 07:59:42] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/04/2022, 07:59:42] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/04/2022, 07:59:42] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/04/2022, 07:59:42] [Tapo Smart Platform] 371 Error: Request failed with status code 429

**Comment by TomKrysz** [comment]
Brand new 1.4.17! :)

**Comment by madmath74** [comment]
Same Log-Message on my HB. 
What I have done so far:

- Created Sub-Bridge for the PlugIn
- Uninstalled the PlugIn, cleared all Files from the HB-Users directory, Reinstalled it and added device per device to the PlugIn again

Still showing the Error message:

[05/11/2022, 10:42:16] [Tapo Smart Platform] Restarting child bridge...
[05/11/2022, 10:42:16] Got SIGTERM, shutting down child bridge process...
[05/11/2022, 10:42:21] [Tapo Smart Platform] Child bridge process ended
[05/11/2022, 10:42:21] [Tapo Smart Platform] Process Ended. Code: 143, Signal: null
[05/11/2022, 10:42:28] [Tapo Smart Platform] Restarting Process...
[05/11/2022, 10:42:29] [Tapo Smart Platform] Launched child bridge with PID 18651
[05/11/2022, 10:42:31] Registering platform 'homebridge-tapo.TapoP100'
[05/11/2022, 10:42:31] [Tapo Smart Platform] Loaded homebridge-tapo v1.4.17 child bridge successfully
[05/11/2022, 10:42:31] Loaded 4 cached accessories from cachedAccessories.0E82BD9C222F.
[05/11/2022, 10:42:31] [Tapo Smart Platform] Loading accessory from cache: Licht in Dianas Büro
[05/11/2022, 10:42:31] [Tapo Smart Platform] Loading accessory from cache: Licht in Michas Büro
[05/11/2022, 10:42:31] [Tapo Smart Platform] Loading accessory from cache: Licht im Schlafzimmer
[05/11/2022, 10:42:31] [Tapo Smart Platform] Loading accessory from cache: Ampel
[05/11/2022, 10:42:31] [Tapo Smart Platform] Restoring existing accessory from cache: Licht in Dianas Büro
[05/11/2022, 10:42:32] [Tapo Smart Platform] Restoring existing accessory from cache: Licht in Michas Büro
[05/11/2022, 10:42:32] [Tapo Smart Platform] Restoring existing accessory from cache: Licht im Schlafzimmer
[05/11/2022, 10:42:32] [Tapo Smart Platform] Restoring existing accessory from cache: Ampel
[05/11/2022, 10:42:33] [Tapo Smart Platform] Adding new accessory: 192.x.x.111
[05/11/2022, 10:42:33] Homebridge v1.5.1 (HAP v0.10.4) (Tapo Smart Platform) is running on port 39654.
[05/11/2022, 10:42:56] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/11/2022, 10:43:04] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/11/2022, 10:43:04] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/11/2022, 10:43:04] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/11/2022, 10:43:08] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[05/11/2022, 10:43:08] [Tapo Smart Platform] 371 Error: Request failed with status code 429

(x-ed out one IP Address)

Don't know, how to fix this.

**Comment by apatsufas** [comment]
Have you played around with these settings? ![image](https://user-images.githubusercontent.com/1316598/203713151-32e70df3-e249-44f3-bc75-73cf72d19da5.jpeg)

**Comment by TomKrysz** [comment]
Of course. I've set interval to 300 secs (5 minutes) and increased timeout value to 180 secs without effects. Unfortunately after couple of minutes flood has begun again.
